src/wowman/core.clj

```diff
@@ -935,25 +941,61 @@
         matching-addon-list (->> addon-list (map find-expand) (remove nil?) vec)]
     matching-addon-list))
 
-(defn import-addon-list
-  "caveats: imports the *latest* version of the addon, if addon found"
+;; v1 takes 13.7 seconds to build an index using the 'short' catalog
+;; this function can still be improved
+(defn import-addon-list-v1
+  "deprecated. handles exports with partial information (name, or name and source) from <=0.10.0 versions of wowman.
+  caveats: imports the *latest* version of the addon, if addon found."
```

Comment:
rm. unless we're handling specific versions of addons then this isn't a caveat, it's an apology